### ISSUE TITLE
Plugins: VoiceOver reads every single plugin listed in each cell and it's hard to follow

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12896]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Plugins.
1. Swipe right to go through each UI element on screen.

**Result:** VoiceOver reads every single plugin listed in the cell. It’s not clear why or what can be done. It would be good to specify just the action, something like ” Installed plugins, double tap to manage installed plugins.” ([44s](https://cloudup.com/cE-jCEr0XdU))

<img src="https://user-images.githubusercontent.com/1119271/68343779-ce12cb80-00aa-11ea-8f4c-2db6edbb1a87.png" width="270" alt="voiceover-plugins">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
